generate/gogen/types.go

```diff
@@ -115,14 +117,14 @@ func (t *goType) isPrimative() bool {
 
 func (t *goType) isNillable() bool {
 	switch t.typeID {
-	case typeBool, typeByte, typeInt32, typeInt64, typeFloat32, typeFloat64, typeString:
+	case typeBool, typeByte, typeInt32, typeInt64, typeFloat32, typeFloat64, typeString, typeClass:
```

Comment:
Is any type still nillable? Should this function be removed?

Reply:
typeByteSlice
	typeSlice
	typeMap
are all still "nlllable"

Reply:
Sort alphabetically

Reply:
Sort alphabetically here still stands